x-pack/plugin/deprecation/src/test/java/org/elasticsearch/xpack/deprecation/ClusterDeprecationChecksTests.java

```diff
@@ -157,7 +157,7 @@ public void testUserAgentEcsCheck() {
             "User-Agent ingest plugin will use ECS-formatted output",
             "https://www.elastic.co/guide/en/elasticsearch/reference/master/breaking-changes-7.0.html" +
                 "#ingest-user-agent-ecs-always",
-            "Ingest pipelines [ecs_false, ecs_null] will change to using ECS output format in 7.0");
+            "Ingest pipelines [ecs_null] will change to using ECS output format in 7.0");
```

Comment:
Do we want to change this to say:
```suggestion
            "Ingest pipelines [ecs_null] will change to using ECS output format by default in 7.0");
```

To make it a bit clearer that you can choose to set `ecs: false` in the pipeline config?

Reply:
++